src/teststand.cpp

```diff
@@ -75,6 +76,9 @@ void Teststand::initialize()
   // Create the joint module objects
   joints_.set_motor_array(motors_, motor_torque_constants_, joint_gear_ratios_,
                           joint_zero_positions_, motor_max_current_);
+
+  // set the max torque we saturate to
+  max_joint_torques_ = 0.99 * joints_.get_max_joint_torques().array();
```

Comment:
Again the `0.99` factor.

If you have a REALLY good reason for using the factor, make it a constant in the class and give it a name at the top of the file or as a static field on the class.

Reply:
Ok, then I will create some static const var